virttest/tests/unattended_install.py

```diff
@@ -1592,6 +1654,21 @@ def copy_images():
                     log_file, rh_upgrade_error_str
                 )
                 post_finish_str_found = string_in_serial_log(log_file, post_finish_str)
+                # For libvirt Windows unattended, also check old log file if available
+                if (
+                    old_log_file
+                    and os.path.exists(old_log_file)
+                    and params.get("vm_type") == "libvirt"
+                    and unattended_install_config.unattended_file
+                    and unattended_install_config.unattended_file.endswith(".xml")
+                ):
+                    try:
+                        if not post_finish_str_found:
+                            post_finish_str_found = string_in_serial_log(
+                                old_log_file, post_finish_str
+                            )
+                    except IOError:
+                        pass
```

Comment:
I think we can change the `log_file` data type to the `list` and store the `serial_console_log` each time the `VM` starts.  For the last serial content checking, we can retrieve the `log_file` list to check the content.
Alternatively, we may append the following serial console content to the original one to keep one serial console log file.
I would like to hear your thoughts. Thanks.

Reply:
Same above comment.